### ISSUE TITLE
Mod Request

### ISSUE THREAD

**Issue by Rainee4563** [Sunekaer/stoneBlock#488]
Ok so I know you might not include this but at least consider it the mod is called cc-tweaks in layman terms  
its a fork of the mod computercraft which contain fixes that is currently in computercraft which may not be fixed due to original author of computercraft no longer working on the mod

**Comment by Sunekaer** [comment]
Sorry are not adding or changing any mods, your welcome to do it on your own install

**Comment by Rainee4563** [comment]
Sorry to be rude but I would like to point out a flaw with that sentence you say your not adding or changing any mods yet if i look in the config of stoneblock you add a mod called building gadgets which after looking up makes repetitive building easier (which you can already do with one of the wands) and even then the current version in the modpack for computercraft when attempting to run a command it spits out an error which i already looked up which can't be fixed
